### ISSUE TITLE
go get fails

### ISSUE THREAD

**Issue by montao** [linuxkit/linuxkit#3216]
**Description**
I could get started with Win 10 Home as host running Ubuntu 18.04 in a Vagrant VM building LinuxKit using make. It worked perfectly. But trying to use go, different versions, I encountered some problems:

**Steps to reproduce the issue:**
vagrant@ubuntu-xenial:~$ go get -u github.com/linuxkit/linuxkit/src/cmd/linuxkit

**Describe the results you received:**
Different error messages depending on which version of go I try it with. 

**Describe the results you expected:**
A clean build. 

**Additional information you deem important (e.g. issue happens only occasionally):**
Using go 1.10 with Windows 10 I don't receive an error message. The error happens with go 1.9, 1.10 and 1.11 with Ubuntu 18.04 running in a Vagrant VM. I am going to try other ways of getting started, like native Ubuntu as host.

**Comment by justincormack** [comment]
What sort of errors are you getting?

**Comment by montao** [comment]
@justincormack The errors were local GoLang compilation errors using an Ubuntu 18.04 native host os. I swtiched computers and then everything worked (but I had to use the switch `--disable-content-trust` when building. Then I could run it (using Win 10 Enterprise as host OS with HyperV as provider and an Ubuntu 18.04 virtual machine run with Vagrant. I will try again with my native Ubuntu to see what is wrong, it looks like it was something wrong with my GoLang or the compiler.
